app/javascript/controllers/file_auth_controller.js

```diff
@@ -32,18 +32,28 @@ export default class extends Controller {
     const manifest = evt.detail
     const canvases = manifest.items
     this.addPostCallbackListener()
-    const resources = canvases.flatMap((canvas) => {
+    this.documents = canvases.flatMap((canvas) => {
       const annotationPages = canvas.items
       return annotationPages.flatMap((annotationPage) => {
         const paintingAnnotations = annotationPage.items.filter((annotation) => annotation.motivation === "painting")
         return paintingAnnotations.map((annotation) => {
           const contentResource = annotation.body
-          this.maybeDrawContentResource(contentResource)
           return contentResource
         })
       })
     })
-    this.firstFile = resources[0]['id']
+    console.log("Resources: ", this.documents)
```

Comment:
Is this console statement here for a specific reason

Reply:
Just for understanding. I can remove it.